src/Superstructure/ESMFMod/include/ESMC_Init.h

```diff
@@ -125,6 +126,29 @@ extern "C" {
 } // extern "C"
 #endif
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+//-----------------------------------------------------------------------------
+//BOP
+// !IROUTINE: ESMC_FinalizeMPI - Finalize the ESMF Framework and specify whether
+//                               MPI should be finalized
+//
+// !INTERFACE:
```

Comment:
If it's possible to thread an enumerated set of options further up the stack rather than a boolean, that might be good too.

Reply:
Added in [6716d2e](https://github.com/esmf-org/esmf/pull/234/commits/6716d2e82e50a51b5e75f43f4a75dd2090fdd8b8).